asm/istio/expansion/vm-eastwest-gateway.yaml

```diff
@@ -45,3 +47,8 @@ spec:
               - name: tls-webhook
                 port: 15017
                 targetPort: 15017
+  values:
+    gateways:
+      istio-ingressgateway:
+        # Enable gateway injection
+        injectionTemplate: gateway
```

Comment:
This doesn't look related to staging images, was this included on purpose?

Reply:
Yes. This is added to use the injection based gateway so that 1.10 eastwest gateway can be created.